projects/BFDR.CLI/README.md

```diff
@@ -0,0 +1,143 @@
+# bfdr-cli
+
+This project contains a small .NET (C#) command-line interface program that demonstrates usage of the BFDR project.
+
+## Usage: Birth Examples
+
+```bash
+# Prints CLI Help
+dotnet run --project BFDR.CLI help
+
+# Builds a fake birth record and print out the record as FHIR XML and JSON
+dotnet run --project BFDR.CLI fakerecord birth
```

Comment:
Can we remove `--project BFDR.CLI` from all the examples and state something like:
> These commands are assumed to run from `projects/BFDR.CLI/`. If running from another folder do `dotnet run --project path/to/BFDR.CLI ...`